docs/pages/machine-workload-identity/mwi-use-cases/mwi-ci-cd.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: CI/CD with Machine & Workload Identity
+description: Configuring CI/CD with Machine & Workload Identity
+labels:
+ - conceptual
+ - mwi
+---
+
+Teleport Machine & Workload Identity eliminates the need for long-lived static secrets in CI/CD pipelines by issuing short-lived, automatically generated certificates at runtime.
+Teleport natively integrates with many CI/CD providers and deployment targets. Refer to the links at the bottom of this page for instructions on using Machine & Workload Identity with your preferred tools.
+
+## Choose your CI/CD platform
+
+<TileGrid
+  tiles={[
+    {
+      icon: <Icon name="azureDevops" size="xl" />,
+      to: "/machine-workload-identity/machine-id/deployment/azure-devops",
+      name: "Azure DevOps",
+    },
+    {
+      icon: <Icon name="bitbucket" size="xl" />,
+      to: "/machine-workload-identity/machine-id/deployment/bitbucket",
+      name: "BitBucket",
+    },
+    {
+      icon: <Icon name="circleci" size="xl" />,
+      to: "/machine-workload-identity/machine-id/deployment/circleci",
+      name: "CircleCI",
+    },
+    {
+      icon: <Icon name="gitlab" size="xl" />,
+      to: "/machine-workload-identity/machine-id/deployment/gitlab",
+      name: "GitLab CI",
+    },
+    {
+      icon: <Icon name="jenkins" size="xl" />,
+      to: "/machine-workload-identity/machine-id/deployment/jenkins",
+      name: "Jenkins",
+    }
+  ]}
+/>
+
+## Eliminate secrets from your CI/CD pipeline
+
+In a typical CI/CD setup, a pipeline such as one running in GitHub Actions on GitHub's infrastructure outside the corporate network builds a 
+container image, pushes it to a container registry, and updates an application running in a container orchestration platform (such as Kubernetes).
+
+This workflow commonly relies on sensitive credentials stored in the CI/CD system, such as:
+
+- Cloud provider access credentials (e.g., AWS, GCP, Azure)
+- Configuration files or tokens for deploying to orchestration platforms
+
+Best practices recommend rotating these secrets every six months. A platform team generates new credentials and shares them with a development team, 
```

Comment:
Do we have a source for the six months claim? I might consider rephrasing this, because we'd probably consider this type of secret not to be a best practice at all.